src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1591941085.inc.php

```diff
@@ -0,0 +1,76 @@
+<h1>Build #1591941085</h1>
+<h2>Date: 2020-06-12</h2>
+<div class="changelog">
+    - #690: Show menu entry for table; add new field type for this
+</div>
+<?php
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_type', 'en')
+    ->setFields([
+        '049_trans' => 'Sidebar menu entry lookup',
+        'force_auto_increment' => '0',
+        'constname' => 'CMSFIELD_SIDEBAR_LOOKUP',
+        'mysql_type' => '',
+        'length_set' => '',
+        'base_type' => 'standard',
+        'help_text' => '',
+        'mysql_standard_value' => '',
+        'fieldclass' => 'ChameleonSystem\CoreBundle\Field\FieldSidebarConnected',
+        'contains_images' => '0',
+        'indextype' => 'none',
+        'class_subtype' => '',
+        'class_type' => 'Core',
+        'id' => 'c15bdac0-e953-68f2-0294-e8a60345a698',
+    ])
+;
+TCMSLogChange::insert(__LINE__, $data);
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_type', 'de')
+    ->setFields([
+        '049_trans' => 'Menü-Eintrag Lookup',
+    ])
+    ->setWhereEquals([
+        'id' => 'c15bdac0-e953-68f2-0294-e8a60345a698',
+    ])
+;
+TCMSLogChange::update(__LINE__, $data);
+
+$data = TCMSLogChange::createMigrationQueryData('cms_field_conf', 'en')
+->setFields([
+    'name' => 'shown_in_menu_as',
+    'translation' => 'Found in the menu as',
+    'cms_field_type_id' => TCMSLogChange::GetFieldType('CMSFIELD_SIDEBAR_LOOKUP'),
+    'cms_tbl_field_tab' => '',
+    'isrequired' => '0',
+    'fieldclass' => 'ChameleonSystem\CoreBundle\Field\FieldSidebarConnected',
+    'modifier' => 'readonly',
```

Comment:
```suggestion
    'modifier' => 'standard',
```

I think the default is "standard", but please check before using my suggestion.
Instead of manually setting the view to readonly, we need to use the readonly view in GetHMTL of the field instead.

Reply:
The default value of `cms_field_conf`.`modifier` is `none` (is changed by this), and the mentioned method is added as suggested by you.